### PR TITLE
Remove instance position from SqsReadConfiguration

### PR DESCRIPTION
**Pull request by adammorr**
It feels to me like instance position was an internal concern prior to releasing JustSaying as an open source project so this PR is to 🔥 it

### CODE REVIEW

**Comment by adammorr** [comment]
Yeah I figured as much. I'll have a go at rationalising the SNS/SQS configuration in a follow up PR

**Comment by adammorr** [comment]
I'll also fix that test...

File: tests/JustSaying.IntegrationTests/Fluent/Publishing/WhenPublishingWithoutAMonitor.cs [hunk 1]
```diff
@@ -49,10 +49,10 @@ public async Task A_Message_Can_Still_Be_Published_To_A_Topic()
                         (config) => config.WithPublishFailureBackoff(TimeSpan.FromMilliseconds(1))
                                           .WithPublishFailureReattempts(1));
 
+                    ;
```
Comment (martincostello) [inline tests/JustSaying.IntegrationTests/Fluent/Publishing/WhenPublishingWithoutAMonitor.cs]: ?

Comment (adammorr) [reply tests/JustSaying.IntegrationTests/Fluent/Publishing/WhenPublishingWithoutAMonitor.cs]: 🤦‍♂ I'm blaming Rider

**Pull request merged**